scripts/ai/AIDriveStrategyUnloadCombine.lua

```diff
@@ -712,40 +758,6 @@ function AIDriveStrategyUnloadCombine:isMoveablePipeDisabled()
     return self.state.properties.moveablePipeDisabled
 end
 
-------------------------------------------------------------------------------------------------------------------------
--- Fill node handling
-------------------------------------------------------------------------------------------------------------------------
--- Make sure the autoAimTargetNode is not moving with the fill level
-function AIDriveStrategyUnloadCombine:fixAutoAimNode()
-    self.autoAimNodeFixed = true
-end
-
--- Release the auto aim target to restore default behaviour
-function AIDriveStrategyUnloadCombine:releaseAutoAimNode()
-    self.autoAimNodeFixed = false
-end
-
-function AIDriveStrategyUnloadCombine:isAutoAimNodeFixed()
-    return self.autoAimNodeFixed
-end
-
--- Make sure the autoAimTargetNode is not moving with the fill level (which adds realism trying to
--- distribute the load more evenly in the trailer but makes life difficult for us)
--- TODO: instead of turning it off completely, could try to reduce the range it is adjusted
-function AIDriveStrategyUnloadCombine:updateFillUnitAutoAimTarget(superFunc, fillUnit)
```

Comment:
I was thinking of moving this to a specialization and limit the movement but what you did is a lot simpler.